### PR TITLE
Update docs for new IIS limits

### PR DESCRIPTION
**Pull request by jkotalik**
Updates regarding new options types we added in IIS inprocess : https://github.com/aspnet/AspNetCore/blob/master/src/Servers/IIS/IIS/src/IISServerOptions.cs

Follow up for https://github.com/aspnet/AspNetCore/issues/8293.

### CODE REVIEW

File: aspnetcore/host-and-deploy/iis/index.md [hunk 1]
```diff
@@ -157,6 +157,8 @@ services.Configure<IISServerOptions>(options =>
...
 | `AuthenticationDisplayName`    | `null`  | Sets the display name shown to users on login pages. |
+| `AllowSynchronousIO `          | `false` | Whether synchronous IO is allowed for the `HttpContext.Request` and the `HttpContext.Response`. |
+| `MaxRequestBodySize`           | `30000000`  | Gets or sets the the max request body size for the `HttpRequest`. Note that IIS itself has the limit `maxAllowedContentLength` which will be processed before the `MaxRequestBodySize` set in the `IISServerOptions`. Changing the `MaxRequestBodySize` will not affect the `maxAllowedContentLength`. To increase `maxAllowedContentLength`, add an entry in the web.config to set `maxAllowedContentLength` to a higher value. See `https://docs.microsoft.com/en-us/iis/configuration/system.webServer/security/requestFiltering/requestLimits/#005` for more details.|
```
Comment (scottaddie) [inline aspnetcore/host-and-deploy/iis/index.md]:
```suggestion
| `MaxRequestBodySize`           | `30000000`  | Gets or sets the the max request body size for the `HttpRequest`. Note that IIS itself has the limit `maxAllowedContentLength` which will be processed before the `MaxRequestBodySize` set in the `IISServerOptions`. Changing the `MaxRequestBodySize` won't affect the `maxAllowedContentLength`. To increase `maxAllowedContentLength`, add an entry in the *web.config* to set `maxAllowedContentLength` to a higher value. For more details, see [Configuration](/iis/configuration/system.webServer/security/requestFiltering/requestLimits/#configuration).|
```

**Comment by guardrex** [comment]
Isn't timing (thus versioning; i.e., 2.1 vs. 2.2+) relevant for the updates?

File: aspnetcore/host-and-deploy/iis/index.md [hunk 2]
```diff
@@ -157,6 +157,11 @@ services.Configure<IISServerOptions>(options =>
 | ------------------------------ | :-----: | ------- |
 | `AutomaticAuthentication`      | `true`  | If `true`, IIS Server sets the `HttpContext.User` authenticated by [Windows Authentication](xref:security/authentication/windowsauth). If `false`, the server only provides an identity for `HttpContext.User` and responds to challenges when explicitly requested by the `AuthenticationScheme`. Windows Authentication must be enabled in IIS for `AutomaticAuthentication` to function. For more information, see [Windows Authentication](xref:security/authentication/windowsauth). |
 | `AuthenticationDisplayName`    | `null`  | Sets the display name shown to users on login pages. |
+::: moniker range=">= aspnetcore-3.0"
```
Comment (guardrex) [inline aspnetcore/host-and-deploy/iis/index.md]: This is going to break the table. The whole table needs to be placed in versioned tags.

File: aspnetcore/host-and-deploy/iis/index.md [hunk 3]
```diff
@@ -153,16 +153,26 @@ services.Configure<IISServerOptions>(options =>
...
 | `AuthenticationDisplayName`    | `null`  | Sets the display name shown to users on login pages. |
-::: moniker range=">= aspnetcore-3.0"	
 | `AllowSynchronousIO`           | `false` | Whether synchronous IO is allowed for the `HttpContext.Request` and the `HttpContext.Response`. |
 | `MaxRequestBodySize`           | `30000000`  | Gets or sets the the max request body size for the `HttpRequest`. Note that IIS itself has the limit `maxAllowedContentLength` which will be processed before the `MaxRequestBodySize` set in the `IISServerOptions`. Changing the `MaxRequestBodySize` won't affect the `maxAllowedContentLength`. To increase `maxAllowedContentLength`, add an entry in the *web.config* to set `maxAllowedContentLength` to a higher value. For more details, see [Configuration](/iis/configuration/system.webServer/security/requestFiltering/requestLimits/#configuration).|
 
 ::: moniker-end
 
+::: moniker range="= aspnetcore-2.2"
```
Comment (guardrex) [inline aspnetcore/host-and-deploy/iis/index.md]:
```suggestion
::: moniker range="< aspnetcore-3.0"
```

**Comment by guardrex** [comment]
ok ... that should work 🤞🍀.

If the build comes back clean, :shipit:.

**Comment by guardrex** [comment]
Dog'gone IT!

> MonikerRange have some invalid chars in the starting.

*Stand-by!!!!!!*

**Comment by guardrex** [comment]
😅 *Finally!!!!* :smile:

**Pull request merged**